tests/test_examples.py

```diff
@@ -100,6 +100,8 @@ def runTest(self):
         import docs.examples.ex13 as ex
         u = ex.u
         A = ex.A
+        current = ex.current
+        self.assertAlmostEqual(current['ground'], -0.441415756775655)
```

Comment:
By this, I mean that both tests should use either the explicit number or the analytical formula. There are advantages to both, the number noticing even the smallest changes (although this can be too sensitive, e.g. across operating systems, BLAS-implementations, hardware, I don't know), the formula reassuring the engineer that the finite element model is doing the right thing. Having the two different bases for the tests is confusing.